source/containers/Form/Form.tsx

```diff
@@ -100,6 +100,7 @@ const Form: React.FC<Props> = ({
         validateStepAnswers={validateStepAnswers}
         status={status}
         questions={questions}
+        allQuestions={formState.allQuestions}
```

Comment:
Whats the difference between allQuestions and questions?

Reply:
questions is the questions belonging to the step. allQuestions is all questions in the form. Which feels a bit bad to have to pass through, but...

Reply:
Då skulle jag döpa om allQuestions till formQuestions resp. stepQuestions